.github/dependabot.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+updates:
+  - package-ecosystem: nuget
+    directory: "/"
+    schedule:
+      interval: daily
+      time: "11:00"
+    open-pull-requests-limit: 5
+    commit-message:
+      prefix: fix
```

Comment:
Why prefix as `fix`?
In general, we use `chore(deps)`

Same for the other ecosystem

Reply:
In [hello-microservice](https://github.com/FromDoppler/hello-microservice) we decided to use `fix` in order to increment the version when a dependency is updated.

Thoughts?